src/Symfony/Component/Form/Extension/Core/Type/ChoiceType.php

```diff
@@ -328,6 +328,8 @@ public function configureOptions(OptionsResolver $resolver)
             'placeholder' => $placeholderDefault,
             'error_bubbling' => false,
             'compound' => $compound,
+            // submitted arrays are dealt with in data transformers
+            'accept_multiple_values' => true,
```

Comment:
Not sure if that's the best name. I could also imagine something like `accept_array_data`.

Reply:
Suggestion: `accept_unstructured_data`?

Reply:
`'scalar' => false`?